eppoclient/assignmentlogger.go

```diff
@@ -6,6 +6,12 @@ type IAssignmentLogger interface {
 	LogAssignment(event AssignmentEvent)
 }
 
+// BanditActionLogger is going to be merged into IAssignmentLogger in
+// the next major version.
+type BanditActionLogger interface {
+	LogBanditAction(event BanditEvent)
+}
```

Comment:
This is to limit upgrade to a minor version bump. If we add `LogBanditAction` to `IAssignmentLogger` that would require a major bump.

See https://go.dev/blog/module-compatibility > working with interfaces